sycl/test-e2e/KernelAndProgram/persistent-cache-multi-device.cpp

```diff
@@ -9,6 +9,9 @@
 // XFAIL: spirv-backend && run-mode
 // XFAIL-TRACKER: CMPLRLLVM-64705
 
+// XFAIL: linux && arch-intel_gpu_bmg_g21 && !igc-dev && run-mode
```

Comment:
Weird that these tests are failing due to GPU reset.

Reply:
yeah its consistent too, i have absolutely no idea whats going on

Reply:
Let me look into this.